src/test/java/bio/terra/service/filedata/flight/ingest/IngestFilePrimaryDataStepTest.java

```diff
@@ -51,20 +46,18 @@ public class IngestFilePrimaryDataStepTest extends TestCase {
           .sourcePath("gs://bucket/path/file.txt")
           .targetPath("/foo/bar/baz.txt");
 
-  @MockBean private GcsPdao gcsPdao;
-
-  @MockBean private Dataset dataset;
+  @Mock private GcsPdao gcsPdao;
 
-  @MockBean private ConfigurationService configService;
+  @Mock private Dataset dataset;
```

Comment:
why switch from mockBean to mock? Is this to avoid SpringRunner?

Reply:
Sort of to avoid, yes. Normally a spring runner test requires `@MockBean` to both mock an api and provide a bean that replaces the real spring bean (component) from being created and injected. When writing a test that uses a spring context, `@MockBean` is the only way to replace a bean with a mock at object creation time.

The odd thing here is that the code wasn't using spring injection at all so this wasn't needed. It was already creating the step explicitly using `new IngestFilePrimaryDataStep()` and passing the mocked objects to the constructor. So this test was using a spring context but wasn't using `Autowired` so using `MockBean` here wasn't having any effect anyway.